docs/README.md

```diff
@@ -1,9 +1,56 @@
-## README
+# README
 
 Welcome to BloodConnect README site. Please follow the sitemap for detailed information and guidelines.
 
-### Development
-- [System Architecture](https://craftsmenltd.github.io/BloodConnect/development/Architecture.html)
-- [Ticket Lifecycle](https://craftsmenltd.github.io/BloodConnect/development/TicketLifecycle.html)
-- [Coding Guideline](https://craftsmenltd.github.io/BloodConnect/development/CodingGuideline.html)
-- [PR and Related Guideline](https://craftsmenltd.github.io/BloodConnect/development/Architecture.html)
\ No newline at end of file
+## Documentation Guideline
+For BloodConnect documentation is being [reStructuredText](https://en.wikipedia.org/wiki/ReStructuredText) (RST, ReST or ReST)
+with [Sphinx](https://www.sphinx-doc.org/en/master/). There are few reasons behind it:
+
+- It is lightweight and better for technical documentation rather than Markdown
```

Comment:
keeping in alignment of the language in the following bullet points, i suggest rephrasing to:
```
- Lightweight and better for technical documentation rather than Markdown`.
- Generation of output for multiple types of documentations (HTML, PDF, LaTeX etc.).
- Math formula and better support for code snippets.
- Built in support of using extensions
```
again, just a suggestion, feel free to disagree